src/include/pbs_db.h

```diff
@@ -754,7 +754,9 @@ int pbs_shutdown_db_async(char **errmsg);
  *			Caller should free this memory after usage.
  *
  */
+#ifndef pbs_get_dataservice_usr
```

Comment:
as you can see i have made pbs_get_dataservice_usr as macro for windows since its of no use there, I had to gaurd this to get the appveyor build passed

Reply:
This guard is to make successful build.